File: 268-missing-number/missing-number.js
var missingNumber = function(nums) {
    let l = nums.length + 1;
    const sum_nums = nums.reduce((acc,curval) => acc + curval,0);
    let sum_og = 0;
    for(let i = 0;i <l;i++){
        sum_og += i;
    }
    return  sum_og - sum_nums;
};

File: 268-missing-number/missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int sum_nums = 0;
        int sum_og = 0;
        for (int i = 0;i <nums.length; i++){
            sum_nums += nums[i];
        }
        for (int j = 0; j < nums.length+1;j++){
            sum_og += j;
        }
        return sum_og - sum_nums;
    }
}